components/emoji_picker/emoji_picker.jsx

```diff
@@ -178,15 +187,24 @@ export default class EmojiPicker extends React.PureComponent {
     componentDidMount() {
         // Delay taking focus because this briefly renders offscreen when using an Overlay
         // so focusing it immediately on mount can cause weird scrolling
-        requestAnimationFrame(() => {
+        window.requestAnimationFrame(() => {
             if (this.searchInput) {
                 this.searchInput.focus();
             }
+            this.renderAllCategoriesFrame = window.requestAnimationFrame(() => {
+                this.renderAllCategories();
+            });
         });
-        this.divHeight = this.emojiPickerContainer.offsetHeight;
+
+        if (this.emojiPickerContainer) {
+            this.divHeight = this.emojiPickerContainer.offsetHeight;
+        }
     }
 
     UNSAFE_componentWillUpdate(nextProps, nextState) { // eslint-disable-line camelcase
+        if (this.state.categories !== nextState.categories) {
```

Comment:
nothing to do now on this PR but we should consider moving this to `componentDidUpdate`